brianlong/www.validators.app
# 182138460 add filters to data centers

#### What's this PR do?
- add new filters for ans and data centers pages.

#### How should this be manually tested?
- go to data_center show page
- see if the filtering buttons work as expected
- go to asn show page
- see if the filtering buttons work as expected
- check the mobile view
- run tests

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182138460)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
